my-app/src/App.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import logo from "./logo.svg";
+import "./App.css";
+
+import Recipe from "./components/Recipe";
+import PrivateRoute from "./components/PrivateRoute";
+import {
+  BrowserRouter as Router,
+  Route,
+  useHistory,
+  Switch,
```

Comment:
If you add `exact` to line 26 you don't need to import switch, you aren't using it right now anyways.